lib/need_for_type/states/menu_state.rb

```diff
@@ -0,0 +1,20 @@
+module NeedForType::States
+  class MenuState < State
+
+    def initialize(display_window)
+      super(display_window)
+    end
+
+    def input_worker(number_option)
```

Comment:
Being picky 😝, `number_option` can be replaced with `total_menu_options` or something like this to be more clear.